src/Select/Option.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import cx from 'classnames'
+import styles from './Option.styles.js'
+
+const createOnClickHandler = (onClick, option) => evt => {
+    evt.stopPropagation()
+    onClick(option)
+}
+
+const Option = ({ value, label, active, disabled, onClick }) => {
+    const handleClicks = createOnClickHandler(onClick, { value, label })
+
+    return (
+        <li
```

Comment:
This (and `ul` in `SelectMenuList`) enforces children to be `li`, so any other custom option component needs to be a `li` as well. I'm not generally opposed to this, just want to raise awareness that it might be cumbersome in some situations, and I don't think the list semantics are needed here (a normal select would expose these semantics either)

Reply:
Yeah. It's because it's based on the Menu component, so they're not yet all conscious decisions. I already took some things out that didn't apply, but there's probably still things in there that don't apply to the Select. We'll have to flesh out what we need here, after we have more clarity on the multi select and the filtered select, and then discuss this with regards to the custom components:

I think for the custom option components we can require two things from users:

1. Adhere to the existing option component api (accept a label, value, onClick prop, etc.)
2. Be a valid SelectMenuList (or whatever ends up being the parent) child

We'll have to decide what we'll want exactly for 2.